README.md

```diff
@@ -19,19 +19,24 @@ module "admin_tier" {
   github_team                 = "${var.github_team}"
   instance_type               = "${var.instance_type}"
   vpc_id                      = "${var.vpc_id}"
-  security_groups             = ["${var.security_groups}"]
+  name                        = "admin"
+  namespace                   = "${var.namespace}"
+  stage                       = "${var.stage}"
   subnets                     = ["${var.subnets}"]
-  associate_public_ip_address = "${var.associate_public_ip_address}"
```

Comment:
@comeanother It is a default value (`associate_public_ip_address=true`) for `tf_admin` module and it is correct I think. But if `tf_instance` has the same default value - it is wrong. Instances should have this parameter set up in `false` by default. So I think we should leave here this parameter as is, but create an issue to fix it in `tf_instance` module